CHANGELOG.md

```diff
@@ -116,6 +116,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Removed
 
```

Comment:
- This needs to be moved to the unreleased section
- This needs to reference the full package path of `otelgrpc`
- This needs to include a PR reference